tests/Google/Ads/GoogleAds/Util/V10/ResourceNamesTest.php

```diff
@@ -108,6 +108,20 @@
  */
 class ResourceNamesTest extends TestCase
 {
+    /**
+     * @covers \Google\Ads\GoogleAds\Util\V10\ResourceNames::forAudience()
+     */
+    public function testGetNameForAudience()
```

Comment:
@fiboknacky Did you add this manually? If so, did you update our code generator so that this will be generated in the next major version?